rudder-web/src/main/scala/bootstrap/liftweb/AppConfig.scala

```diff
@@ -1721,28 +1707,21 @@ object RudderConfig extends Loggable {
         , asyncDeploymentAgent
         , uuidGen
       )
-    , new CheckSystemGroups (
-          rudderDitImpl
-        , roLdap
-        , ldapEntityMapper
-        , groupLibReadWriteMutex
-        , woNodeGroupRepository
-        , uuidGen
-      )
     , new ResumePolicyUpdateRunning(
           asyncDeploymentAgent
         , uuidGen
       )
+    , new CheckCfengineSystemRuleTargets(rwLdap)
+    , new CheckDSCSystemPolicy(rwLdap)
```

Comment:
these two lines should have been before the ResumePolicyUpdateRunning, as it states that it should be after everything